File: src/gestorAplicacion/tareas/Nota.java
package gestorAplicacion.tareas;
import java.io.Serializable;

public class Nota implements Serializable{	
	String titulo;
	String decripcion;
	
	public Nota(String titulo, String descripcion){
		this.titulo=titulo;
		this.decripcion=descripcion;
	}
}

File: python/tareas/nota.py
from persistencia.serializacion import Serializador

#os objetos de esta clase siren como anotaciones o puntos a tener en cuenta en las tareas
class nota:
    def __init__(self, titulo, descripcion):
        self.titulo = titulo
        self.descripcion = descripcion
        Serializador.notas.append(self)
        Serializador.guardarNotas(Serializador.notas)

    def getTitulo(self):
        return self.titulo

    def setTitulo(self, titulo):
        self.titulo = titulo

    def getDescripcion(self):
        return self.descripcion

    def setDescripcion(self, descripcion):
        self.descripcion = descripcion